modules/installation-ibm-cloud-config-yaml.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_cloud_public/installing-ibm-cloud-customizations.adoc
+// * installing/installing_ibm_cloud_public/installing-ibm-cloud-network-customizations.adoc
+
+ifeval::["{context}" == "installing-ibm-cloud-network-customizations"]
+:with-networking:
+endif::[]
+ifeval::["{context}" != "installing-ibm-cloud-network-customizations"]
+:without-networking:
+endif::[]
+
+[id="installation-ibm-cloud-config-yaml_{context}"]
+= Sample customized `install-config.yaml` file for IBM Cloud
```

Comment:
No markup in headings, so:
= Sample customized install-config.yaml file for IBM Cloud

Reply:
Done.